src/core/client/admin/components/UserStatus/BanModal.tsx

```diff
@@ -113,7 +113,7 @@ const BanModal: FunctionComponent<Props> = ({
       try {
```

Comment:
It looks like `input` is no longer being used here. Also, it seems like we are no longer using the form input here at all; can we also just remove lines 178-183 setting initial values for these?